### PR TITLE
Add feedback form

### PR DESCRIPTION
**Pull request by AmirMingaliev**
Task: https://www.pivotaltracker.com/story/show/129058683

### CODE REVIEW

File: app/interactors/feedbacks/validate_and_send.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module Feedbacks
+  class ValidateAndSend
+    include Interactor
+
+    delegate :feedback, to: :context
+
+    def call
+      context.fail! if feedback.invalid?
+
+      FeedbackMailer.new_feedback(feedback).deliver_now
```
Comment (ArthurZaharov) [inline app/interactors/feedbacks/validate_and_send.rb]: separate job class?

File: app/controllers/feedbacks_controller.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class FeedbacksController < ApplicationController
+  expose :feedback
+
+  def new
+  end
+
+  def create
+    result = Feedbacks::ValidateAndSend.call(feedback: feedback)
```
Comment (ArthurZaharov) [inline app/controllers/feedbacks_controller.rb]: I prefer Feedbacks::Send

File: app/mailers/application_mailer.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: "from@example.com"
+  default to: ENV["email_to"],
```
Comment (ArthurZaharov) [inline app/mailers/application_mailer.rb]: up case env variables

File: app/views/feedbacks/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<div class="row">
```
Comment (ArthurZaharov) [inline app/views/feedbacks/new.html.erb]: slim?

File: spec/acceptance/feedbacks/create_feedback_spec.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+describe "Create feedback", type: :feature do
+  before { visit new_feedback_path }
+
+  scenario "User creates feedback with valid params" do
+    fill_in "Your name", with: "Homer Simpson"
```
Comment (ArthurZaharov) [inline spec/acceptance/feedbacks/create_feedback_spec.rb]: maybe create factory and use attributes_for?

File: app/controllers/feedbacks_controller.rb [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+    if result.success?
+      redirect_to new_feedback_path, notice: I18n.t("feedback.sended")
```
Comment (AskarZinurov) [inline app/controllers/feedbacks_controller.rb]: Send sent sent

File: config/locales/en.yml [hunk 7]
```diff
@@ -20,4 +20,8 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  mailer:
+    feedback:
+      subject: "New feedback from %{name}"
+  feedback:
+      sended: "Feedback was successfully sended"
```
Comment (AskarZinurov) [inline config/locales/en.yml]: Send sent sent

File: config/locales/en.yml [hunk 8]
```diff
@@ -20,4 +20,8 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  mailer:
+    feedback:
+      subject: "New feedback from %{name}"
+  feedback:
+      sended: "Feedback was successfully sent"
```
Comment (AskarZinurov) [inline config/locales/en.yml]: @AmirMingaliev how your specs are passing?

File: spec/interactors/feedbacks/send_spec.rb [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+    before do
+      allow(FeedbackMailer).to receive(:new_feedback).and_return(mailer)
+
+      described_class.call(feedback: feedback)
+    end
+
+    let(:mailer) { instance_double("ActionMailer::MessageDelivery", deliver_now: true) }
+    let(:feedback) { Feedback.new(feedback_params) }
+
+    context "when feedback is valid" do
```
Comment (AskarZinurov) [inline spec/interactors/feedbacks/send_spec.rb]: Maybe "is not valid"?

**Comment by AskarZinurov** [comment]
:ok:

**Pull request merged**